extensions/ql-vscode/src/view/jest.config.ts

```diff
@@ -171,7 +171,7 @@ const config: Config = {
     "^.+\\.tsx?$": [
       "ts-jest",
       {
-        tsconfig: "src/view/tsconfig.spec.json",
+        tsconfig: "<rootDir>/tsconfig.spec.json",
```

Comment:
Just wondering why does this change need to happen for this PR? Could this be moved to a separate PR so it's not mixed in with unrelated changes?